application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java

```diff
@@ -108,7 +109,7 @@
 @Component
 public class ActorSystemContext {
 
-    protected final ObjectMapper mapper = new ObjectMapper();
+    protected final ObjectMapper mapper = JacksonUtil.OBJECT_MAPPER;
```

Comment:
This is wrong. You should remove this variable and use static methods from JacksonUtil where the "mapper" was used. Same for all other files.